### PR TITLE
Nav2

### PR DESCRIPTION
**Pull request by evilnick**
adds new sections to nav  for new pages around models and controllers
moves a few pages

### CODE REVIEW

**Comment by evilnick** [comment]
fixing per comments in HO

**Comment by evilnick** [comment]
@pmatulis try this

File: src/en/clouds-manual.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 Title: Manual provisioning  
 
+
+!!! Note: This page has not been updated for Juju 2.0 yet
+
 # Manual Provisioning
```
Comment (pmatulis) [inline src/en/clouds-manual.md]: provisioning

File: src/en/clouds-MAAS.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 Title: Configuring for MAAS (bare metal)  
+Todo: Update MAAS instructions
+
+!!! NOTE: This page has not been updated for Juju 2.0 
 
 
 # Configuring for MAAS (bare metal)
```
Comment (pmatulis) [inline src/en/clouds-MAAS.md]: MAAS can be used with virtual machines but ok

**Comment by pmatulis** [comment]
Review complete

**Comment by evilnick** [comment]
Cool. I am not going to fix the text on those moved pages as they need to be rewritten anyway

**Pull request merged**